chrome/extension/inject.js

```diff
@@ -4,18 +4,16 @@ function updateStyles() {
     (request, sender) => {
       if (!sender.tab) {
         if (request.update === 'style') {
-          const property = request.property;
-          const el = document.querySelectorAll(request.selector);
-          const color = request.color;
+          const { property, selector, color } = request;
+          const el = document.querySelectorAll(selector);
           let rgba;
 
           if (color) {
             rgba = `rgba(${color.r},${color.g},${color.b},${color.a})`;
           }
 
-          el.forEach(target => {
-            const element = target;
-            if (request.color) {
```

Comment:
I simply changed the above assignments because it was inconsistent that `const color = request.color` had been assigned and then we call `request.color` here in line 18. And then I just decided to put everything into the ES6 (or is this ES7?) style object assignment. (and then also removed the parameter renaming but that's related to the eslint rule I discussed above).

I guess to make everything consistent we should use the same kind of syntax in the other functions of inject.js, I can change that for consistency quickly if people agree with this style.